src/commands/helpers/scheduleValidators.ts

```diff
@@ -0,0 +1,26 @@
+import { TextChannel, DMChannel, NewsChannel } from "discord.js";
+
+export const days: Record<string, number> = {
+  Sunday: 0,
+  Monday: 1,
+  Tuesday: 2,
+  Wednesday: 3,
+  Thursday: 4,
+  Friday: 5,
+  Saturday: 6,
+};
+
+export type Channel = TextChannel | DMChannel | NewsChannel;
+
+export const validate = {
```

Comment:
I probably wouldn't export as an object. You can just export the functions themselves, and then if you want an object, you can then do `import * as validate from "./helpers/scheduleValidators";`